doc/help/elektra-data-structures.md

```diff
@@ -377,26 +377,7 @@ if the memory usage gets under a quarter.
 
 The data will be stored as void pointer and can be inserted with the push call.
 Retrieve is possible through the pop call.
-Besides the classical init and del functions, an is empty check is also implemented.
-
-The typedef `Vstack` contains the following fields:
-
-	typedef struct
-	{
-		size_t minSize;
-		size_t size;
-		void ** data;
-		void ** head;
-	} Vstack;
-
-- **minSize**: Is the minimum size of the `Vstack`. After each pop a check is performed to see if a shrink is needed.
-This field defines the minimum size, which the `Vstack` size will never get below.
-
-- **size**: Is the actual allocated size of the `Vstack`
-
-- **data**: Is the array in which the data will be stored.
-
-- **head**: Is the stack pointer for fast insertion and retrieve. After each push or pop the stack pointer gets incremented or decremented.
+Besides the classical init and del functions, an is-empty check is also implemented.
```

Comment:
Yes, please list all relevant source material that has to do with the docu. Going in both directions might be useful, too. Would be great if doxygen can make actual links out if it.

You can think of this docu as requirements and creating links to other artefacts as described in https://en.wikipedia.org/wiki/Requirements_traceability (I suppose you learned that in some requirements lecture already.)